lib/pure/collections/sets.nim

```diff
@@ -516,6 +516,22 @@ proc `-+-`*[A](s1, s2: HashSet[A]): HashSet[A] {.inline.} =
   ## Alias for `symmetricDifference(s1, s2) <#symmetricDifference>`_.
   result = symmetricDifference(s1, s2)
 
+proc `+=`*[A](s1: var HashSet, s2: HashSet[A]) {.inline.} =
+  ## Alias for `s1 = union(s1, s2) <#union>`_.
+  s1 = union(s1, s2)
+
+proc `*=`*[A](s1: var HashSet, s2: HashSet[A]) {.inline.} =
+  ## Alias for `s1 = intersection(s1, s2) <#intersection>`_.
+  s1 = intersection(s1, s2)
+
+proc `-=`*[A](s1: var HashSet, s2: HashSet[A]) {.inline.} =
+  ## Alias for `s1 = difference(s1, s2) <#difference>`_.
+  s1 = difference(s1, s2)
+
+proc `-+-=`*[A](s1: var HashSet, s2: HashSet[A]) {.inline.} =
```

Comment:
I've never used "symmetric set difference" in my entire life. Can we not introduce pointless shortcuts please? Consistency is not the only criterion for a design. Another one is "bloat".

Reply:
After just 3 months, I addressed this :)